stylesheets/_structure.type.scss

```diff
@@ -114,3 +114,8 @@ dt {
         font-size: 2em;
     }
 }
+
+body a:not(.btn).is-disabled {
```

Comment:
Is `body` necessary if you're making the colour immutable? (guessing it was added for higher specificity?)